packages/forest-cloud/src/services/starting-agent-locally.ts

```diff
@@ -0,0 +1,114 @@
+import { AgentOptions, createAgent } from '@forestadmin/agent';
+import {
+  ConnectionParams,
+  MongoDatasourceParams,
+  createMongoDataSource,
+} from '@forestadmin/datasource-mongo';
+import { ConnectionOptions, createSqlDataSource } from '@forestadmin/datasource-sql';
+import fs from 'fs/promises';
+
+import { throwIfNoBuiltCode } from './access-file';
+import DistPathManager from './dist-path-manager';
+import loadCustomization from './load-customization';
+import { BusinessError } from '../errors';
+
+type Datasources = Record<
+  string,
+  {
+    connectionOptions: ConnectionOptions | ConnectionParams;
+    datasourceSuffix?: string;
+  }
+>;
+
+// eslint-disable-next-line import/prefer-default-export
+export async function startingAgent(
+  distPathManager: DistPathManager,
+  { forestServerUrl, envSecret },
+  logger: AgentOptions['logger'],
+): Promise<void> {
+  const agentOptions: AgentOptions = {
+    authSecret: 'dev8environment'.repeat(8),
```

Comment:
nothing really but typo maybe ?
```suggestion
    authSecret: 'dev_environment'.repeat(8),
```